fusedrug/data/molecule/smi_character_histogram.py

```diff
@@ -79,7 +79,7 @@ def smi_file_character_histogram_multiprocessed(
         as_iterator=True,
         copy_to_global_storage=dict(indexed_text_file=itf),
     ):
-        total_characters_counter.update(curr_counter)
+        total_characters_counter |= curr_counter
```

Comment:
Although both are equivalent lets stay with `.update()` for better readability by the reader